themes/default/content/docs/guides/saml/gsuite.md

```diff
@@ -59,7 +59,7 @@ URLs of the `acs` and `metadata` and `sso` endpoints of the Pulumi API, adjusted
     > **Important:** Do not change the value of Name ID Format value once your users have started using Pulumi---not even switching its value between `EMAIL` or `PERSISTENT`.
 
 1. The final step---attribute mapping---is optional, but you may wish to specify proper
-first and last names for your Pulumi users, based on their Google account profiles. The Pulumi service
+first and last names for your Pulumi users, based on their Google account profiles. The Pulumi Cloud
```

Comment:
```suggestion
first and last names for your Pulumi users, based on their Google account profiles. Pulumi Cloud
```

Reply:
@jkodroff lets leave this as it was before this pr opened, we can come back later if we establish a standard around the.